styles/components/_showcode.scss

```diff
@@ -0,0 +1,34 @@
+.showCode {
```

Comment:
@bebraw - I want to use styles from here. But I'm not sure how its connected to the jsx file.